haproxy/README.md

```diff
@@ -0,0 +1,5 @@
+# Weighted load balancer using haproxy and ALB
+Edit `variables.tf` and `terraform apply`
+
```

Comment:
Can you add something about any pre-req work required i.e. create a hosted zone?

Reply:
not needed anymore! 😄